Revit_Engine/Revit_Engine.csproj

```
@@ -85,6 +85,10 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Library_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Physical_Engine, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Apologies, one more: reference should contain only the name of the assembly, without version, culture and processor architecture. Unfortunately when you add the reference using Visual Studio, these get filled automatically. The easiest way to clean this up to open the file on a text editor, e.g. Notepad++.
```suggestion
    <Reference Include="Physical_Engine">
```